cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -7,25 +7,32 @@ gen = ParameterGenerator()
 
 damping_method_enum = gen.enum([ 
                        gen.const("manipulability",      int_t, 0, "manipulability"),
-                       gen.const("constant",            int_t, 1, "constant"),
-                       gen.const("truncation",          int_t, 2, "truncation")],
+                       gen.const("constant",            int_t, 1, "constant"),], 
```

Comment:
It is a little confusing because now "No_damping" is selected and in the line below there is a damping factor showing a value != 0.0.